rules-tests/Transform/Rector/MethodCall/MethodCallToMethodCallRector/Source/FirstDependency.php

```diff
@@ -4,7 +4,7 @@
 
 namespace Rector\Tests\Transform\Rector\MethodCall\MethodCallToMethodCallRector\Source;
 
-final class FirstDependency
+class FirstDependency
```

Comment:
this can be reverted.

Reply:
I had no idea this would work as the class is extended in one of the fixtures but apparently, it's ok.